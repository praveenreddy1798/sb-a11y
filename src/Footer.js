import {
  ButtonGroup,
  Container,
  IconButton,
  Stack,
  Text,
} from '@chakra-ui/react';
import * as React from 'react';
import { FaGithub, FaLinkedin, FaTwitter } from 'react-icons/fa';
import { Logo } from './Logo';

// Footer

export const Footer = () => (
  <Container
    maxW="full"
    as="footer"
    bg="gray.100"
    py={{ base: '12', md: '16' }}
  >
    <Stack maxW="container.sm" mx="auto" spacing={{ base: '4', md: '5' }}>
      <Stack justify="space-between" direction="row" align="center">
        <Logo />
        <ButtonGroup variant="ghost" aria-label="Social links">
          <IconButton
            as="a"
            href="https://www.linkedin.com/company/74529112/"
            icon={<FaLinkedin fontSize="1.25rem" />}
            aria-label="LinkedIn"
          />
          <IconButton
            as="a"
            href="https://github.com/storybookjs/"
            icon={<FaGithub fontSize="1.25rem" />}
            aria-label="GitHub"
          />
          <IconButton
            as="a"
            href="https://twitter.com/storybookjs/"
            icon={<FaTwitter fontSize="1.25rem" />}
            aria-label="Twitter"
          />
        </ButtonGroup>
      </Stack>
      <Text fontSize="sm" color="subtle">
        The MIT License (MIT)
      </Text>
    </Stack>
  </Container>
);
