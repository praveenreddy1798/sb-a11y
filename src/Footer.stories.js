import React from 'react';
import { Footer } from './Footer';
// Footer
export default {
  component: Footer,
  title: 'Components/Footer',
};

const Template = (args) => <Footer {...args} />;

export const Default = Template.bind({});
