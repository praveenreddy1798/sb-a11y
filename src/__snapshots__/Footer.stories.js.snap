// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Footer Default smoke-test 1`] = `
Object {
  "checked": undefined,
  "children": Array [
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "Storybook",
      "pressed": undefined,
      "role": "img",
      "value": undefined,
    },
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "LinkedIn",
      "pressed": undefined,
      "role": "link",
      "value": undefined,
    },
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "GitHub",
      "pressed": undefined,
      "role": "link",
      "value": undefined,
    },
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "Twitter",
      "pressed": undefined,
      "role": "link",
      "value": undefined,
    },
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "The MIT License (MIT)",
      "pressed": undefined,
      "role": "text",
      "value": undefined,
    },
  ],
  "name": "components-footer--default",
  "pressed": undefined,
  "role": "WebArea",
  "value": undefined,
}
`;
